SparseGrids/gridtestExternalTests.cpp

```diff
@@ -104,6 +128,23 @@ ExternalTester::ExternalTester(int in_num_mc) : num_mc(in_num_mc), verbose(false
     #ifdef Tasmanian_ENABLE_DPCPP
     test_queue.init_testing();
     #endif
+    // Hardcoded test types.
+    integr_only = {type_integration};
```

Comment:
When long names differ by only two characters, it becomes very hard to read. Try naming those `quad_only` and `interp_only`

Reply:
Changed. I also shorted the other test case names.